config/crd/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
+# It should be run by config/default
+resources:
+- bases/agents.x-k8s.io_sandboxes.yaml
```

Comment:
Personally I always delete all this kustomize boilerplate, just because it makes everything so complicated.  But maybe we commit so it's in the history and then delete